src/csharp/Microsoft.Spark/Utils/AssemblyLoader.cs

```diff
@@ -10,6 +10,46 @@
 
 namespace Microsoft.Spark.Utils
 {
+    internal static class AssemblySearchPathResolver
+    {
+        internal const string AssemblySearchPathsEnvVarName = "DOTNET_ASSEMBLY_SEARCH_PATHS";
+
+        // Returns the paths to search when loading assemblies in the following order of
```

Comment:
Can you update the comments in TryLoadAssembly  ?  You should be able to move the remarks regarding working dir here.

Reply:
Thanks. Updated.